core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -605,14 +646,35 @@ private MethodMetrics getGlobalMetrics(RequestEvent event) {
     }
 
     private MethodMetrics getMethodMetrics(RequestEvent event) {
-      ResourceMethod method = event.getUriInfo().getMatchedResourceMethod();
-      if (method == null) {
+      ResourceMethod method;
+      try {
+        method = event.getUriInfo().getMatchedResourceMethod();
```

Comment:
there is no need to catch npe for this, as getUriInfo doesn't have request/response